chart-app/src/components/ChartApp.vue

```diff
@@ -52,8 +53,18 @@ export default {
                 chart2: {label: 'Chart 2'},
                 chart3: {label: 'Chart 3'},
             },
+            key: '',
         }
     },
+    methods: {
+        revealChart(event) {
+            if (event.target.value === 'chart1') {
+                document.querySelector('#chart-container').removeAttribute('hidden')
+            } else {
+                document.querySelector('#chart-container').setAttribute('hidden', 'true')
+            }
+        },
+    },
```

Comment:
```suggestion
    computed: {
        hideChart() {
            return this.activeChart !== 'chart1'
        }
    },
```

As a side note - In Vue, if you want to manipulate a DOM element with JS like this, it's generally preferred to use refs. e.g.:
```
<div ref="horse">Silence..</div>
...
horseAround() {
    this.$refs.horse.innerText = "naaay"
}
```

Reply:
I think manipulating the DOM directly is generally not recommended, so `$refs` should only be used when there aren't better ways to solve the problem.